lib/ansible/modules/windows/win_security_policy.py

```diff
@@ -49,6 +39,16 @@
     - The value for the ini key or policy name.
     - If the key takes in a boolean value then 0 = False and 1 = True.
     required: yes
+notes:
+- This module uses the SecEdit.exe tool to configure the values, more details
+  of the areas and keys that can be configured can be found here
+  U(https://msdn.microsoft.com/en-us/library/bb742512.aspx).
+- If you are in a domain environment these policies may be set by a GPO policy,
+  this module can temporarily change these values but the GPO will override
+  it if the value differs.
+- You can also run C(SecEdit.exe /export /cfg C:\temp\output.ini) to view the
+  current policies set on your system.
+- When assigning user rights, use the M(win_user_right) module instead.
```

Comment:
Probably want a seealso for `win_user_right`

Reply:
Good catch. Fixed!